### PR TITLE
[fix bug 1363802] /new variation to support ad campaign - BATM

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds another new ad campaign download page variation

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1363802
https://bugzilla.mozilla.org/show_bug.cgi?id=1364988

## Testing
- It's a new page, so some basic cross browser testing should cover things?
- Check for copy / pasta?

Demo:
https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=batmfree
https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=batmprivate
https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=batmnimble
https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/?xv=batmresist

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
Marking this as DNM for now as scene2 is still lacking direction, but should be good for review

File: bedrock/firefox/templates/firefox/new/onboarding/scene1.html [hunk 1]
```diff
@@ -136,7 +136,7 @@ <h4 class="heading-windows">{{_('Download Firefox <span>for Windows</span>') }}<
         <h4 class="heading-osx">{{_('Download Firefox <span>for macOS</span>') }}</h4>
         <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
 
-        {{ download_firefox(dom_id='download-other-platforms-modal', alt_copy=_('Free Download')) }}
+        {{ download_firefox(dom_id='download-other-platforms-modal', alt_copy=_('Free Download'), locale_in_transition=True) }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/onboarding/scene1.html]: Not related to this change, but I noticed the dl buttons in the modal weren't including the locale so fixing that here.

File: media/css/firefox/new/break-free.scss [hunk 2]
```diff
@@ -3,7 +3,6 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 @import '../../pebbles/includes/lib';
-@import '../../pebbles/components/buttons-download';
```
Comment (alexgibson) [inline media/css/firefox/new/break-free.scss]: `buttons-download` is now included in global.scss, so they should no longer be needed here.

**Comment by alexgibson** [comment]
OK, this should now be good for review / merge.

**Comment by alexgibson** [comment]
Added a second commit with three new page variations for this theme

File: bedrock/settings/static_media.py [hunk 3]
```diff
@@ -1383,6 +1389,13 @@
         ),
         'output_filename': 'js/firefox_new_scene1_fx_lifestyle-bundle.js',
     },
+    'firefox_new_scene1_batm': {
+        'source_filenames': (
+            'js/base/mozilla-modal.js',
```
Comment (jpetto) [inline bedrock/settings/static_media.py]: Do we need the modal library for these variations?

File: bedrock/firefox/templates/firefox/new/batm/scene1.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block optimizely %}
+  {% if switch('firefox-new-optimizely', ['en-US']) %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/batm/scene1.html]: I'm going to bet that this gets asked for at some point. I *could* take it out though?

File: bedrock/firefox/templates/firefox/new/batm/scene2.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block body_class %}scene-2{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/batm/scene2.html]: Oh, yes we should add it there too, else they can't measure the ad spend. I can do it in this PR.

File: bedrock/settings/static_media.py (hunk 3 above)
Comment (alexgibson) [inline bedrock/settings/static_media.py]: nice catch - this is no longer needed.

**Pull request merged**